pkg/flare/service_windows.go

```diff
@@ -0,0 +1,286 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I would assume that this file's ownership should be @DataDog/windows-agent. I'm guessing maybe even `pkg/flare/archive_win.go`?

Reply:
Hi @sgnn7 yes these files would fall under ownership of @DataDog/windows-agent .

Reply:
I've added @DataDog/windows-agent as the code owners for these files